Britannia/localization/english/event_localization/BTWK_events_l_english.yml

```diff
@@ -1,50 +1,50 @@
 ﻿l_english:
+# Loc yet untested: BTWK_frontier_event.0204_desc_contribution
+
 ############# PROVINCES #############
   ############# FRONTIER DECISIONS ############## 
   BTWK_frontier_event.0001:0 "The Welsh Frontier"  # ROOT of decision trigger
-  BTWK_frontier_event.0001_desc:0 "Mercian dominance over the British Midlands is now unquestioned. Having taken full control of the formerly Powysian & Gwentish heartlands, we have in doing so isolated by land the #italic Wealas#! of the North and of the South. With their separation, any hope of a united front against us has been all but crushed. \n\nHowever, when left unchecked, they still remain to send raiding parties into our border regions. The only way to stop this is to emulate the Romans, and construct an embankment from sea to sea as what was achieved with Hadrian's Wall."
+  BTWK_frontier_event.0001_desc:0 "Mercian dominance over the British Midlands is now unquestioned. Having taken full control of the former heartlands of Lowland Powys and Glevum, we have separated the #italic Wealas#! of the North and of the West. With their separation, any hope of a united front against us has been all but crushed. \n\nHowever, the remaining kingdoms have not abated in their hostility and continue to raid our border regions from their mountainous redoubts. The only way to stop this is to emulate the Romans, and construct an embankment from sea to sea as what was achieved with Hadrian's Wall."
```

Comment:
I wrote that in the context of the whole midlands, not the border regions, which I realise you were probably referring to? In that case I'll probably change it to Pengwern as it does fit better in the theme of creating that defined Welsh border

Reply:
Do you want custom localisation for Pengwern & Glevum so their regions are highlighted on the map or?

Reply:
It's also that Pengwern can be considered to cover basically all of "lowland Powys", i.e. it can also described a region bigger than the current ingame duchy.

I assume you mean title links? That's up to you, but since they'll probably be de jure overwritten by the time a player gets this event I'm not sure it'll work properly.

Reply:
No, I meant 'custom_region's. Depends if we want to bloat the geographical_regions file with minor things like that though

Reply:
Ah okay. If it was something relevant to gameplay (e.g. requirements for some event/decision, or area of some effect) then I'd say go for it, but some flavour event localisation is probably pushing it a little far.